docs/CHANGELOG.md

```diff
@@ -1,5 +1,35 @@
 # Changelog
 
+
+## 1.0.0b1
+
+Feature:
+* Add new arguments to `aql.load_file` - `native_support_kwargs` and `enable_native_fallback`. `native_support_kwargs` is a keyword argument to be used by method involved in native support flow and `enable_native_fallback` can be used to fall back to default transfer. `aql.load_file`, by default, uses native approaches to load datasets into Google BiqQuery and Snowflake. The old behaviour can be re-enabled by setting `use_native_support=False`. [#557](https://github.com/astronomer/astro-sdk/issues/557), [#481](https://github.com/astronomer/astro-sdk/issues/481)
```

Comment:
@sunank200 following what we just talked about:

* Improved the performance of `aql.load_file` by supporting database-specific native methods. This is now the default behaviour. Previously, The Astro SDK Python would always use Pandas to load files to SQL databases. Now, there are three modes:
  i. Native: default, uses Bigquery Load Job in the case of BigQuery and Snowflake COPY INTO using external stage
  ii. Pandas: how datasets were previously loaded (to enable this mode, use the argument XX
  iii. Hybrid: attempt to use the native strategy to load a file to the database and if it fails (i), fallback to Pandas (ii)

Reply:
[Done](https://github.com/astronomer/astro-sdk/pull/576/commits/850c76559f23e121934883691e089807411cc309)